t/lib/WTSI/NPG/HTS/Illumina/AlnDataObjectTest.pm

```diff
@@ -453,7 +453,15 @@ sub header : Test(9) {
         ok($obj->header, "$format header can be read");
 
         # 2 * 2 * 1 tests
-        cmp_ok(scalar @{$obj->header}, '==', 11,
+        my $num_lines = scalar @{$obj->header};
+        $num_lines ||= 0;
+        # Starting from samtools v. 1.10.0, a PG line is inserted
+        # into the header for each samtools invocation. The original
+        # number of header lines is 11, two more samtools operations
+        # are invoked during the test, bringing the number of lines
+        # to 13. With older versions of samtools 11 header lines will
+        # be still present at this point.
+        ok((($num_lines == 13) or ($num_lines == 11)),
```

Comment:
If we specify the samtools version (as we do for baton here https://github.com/wtsi-npg/npg_irods/blob/devel/.travis.yml#L16 and here https://github.com/wtsi-npg/npg_irods/blob/devel/scripts/travis_install.sh#L28) then we know exactly what the answer should be. Thoughts?

Reply:
Having discussed this, we decided to test for a least one @PG header line